packages/esm-bed-management-app/src/bed-administration/bed-administration-form.component.tsx

```diff
@@ -116,6 +113,7 @@ const BedAdministrationForm: React.FC<BedAdministrationFormProps> = ({
   };
 
   return (
+    // TODO: Port this over to the modal system or create individual modals for each form
```

Comment:
I vote for option 2.

Reply:
I've filed https://openmrs.atlassian.net/browse/O3-4199